packages/fast-tooling-react/src/css-editor/border/border.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "CSS Border",
+    "description": "A CSS border component's schema definition.",
+    "id": "@microsoft/fast-tooling-react/css-border",
+    "type": "object",
+    "properties": {
+        "data": {
+            "title": "CSS Values",
+            "type": "object",
+            "properties": {
+                "border": {
```

Comment:
you want to change this to not have `border` but instead have `borderWidth`, `borderStyle`, `borderColor`